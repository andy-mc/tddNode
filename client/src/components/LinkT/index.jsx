import React from "react";

export default function LinkT({address}) {
  const add = (a, b) => a + b;
  return (
    <>
     <h2>HOla</h2>
     <a href={address}>{add(1, 3)}</a>
    </>
  )
}
