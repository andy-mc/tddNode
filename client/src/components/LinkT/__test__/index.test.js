import React from "react";
import renderer from 'react-test-renderer';
import LinkT from '../index.jsx';

it('LinkT', () => {
  const tree = renderer
    .create(<LinkT address="google.com"></LinkT>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

