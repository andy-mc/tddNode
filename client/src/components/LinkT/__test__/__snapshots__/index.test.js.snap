// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkT 1`] = `
Array [
  <h2>
    HOla
  </h2>,
  <a
    href="google.com"
  >
    4
  </a>,
]
`;
